ref/general/mp-openapi.adoc

```diff
@@ -47,7 +57,10 @@ For example, the following code example shows JAX-RS code to request the list of
     }
 ----
 
-In the following OpenAPI documentation example that is generated from the example code, the JAX-RS framework generates the path /inventory/systems path based on the value of the @Path annotation, and the value of the operationId key (listContents) based on the name of the method (listContents()). The other parts of the document are generated from the OpenAPI annotations that the developer annotated the code with. For example, the @Operation annotation provides more meaningful information about what the operation does and why; in this case, the operation returns a list of the host names with properties of each host that are currently stored by the inventory web service. This is described by the summary and description keys in the @Operation annotation:
+In the following OpenAPI documentation example that is generated from the example code, the JAX-RS framework generates the `path /inventory/systems` path based on the value of the `@Path` annotation, and the value of the `operationId` key (listContents) based on the name of the method (listContents()).
+The other parts of the document are generated from the OpenAPI annotations that the developer annotated the code with.
+For example, the `@Operation` annotation provides more meaningful information about what the operation does and why; in this case, the operation returns a list of the host names with properties of each host that are currently stored by the inventory web service.
+This is described by the summary and description keys in the `@Operation` annotation:
```

Comment:
Change "This" to reference what is being described to avoid ambiguity. So maybe change it something like "This action" or whatever is being described by the summary and description keys.